sklearn/utils/extmath.py

```diff
@@ -600,6 +600,7 @@ def log_logistic(X, out=None):
     """
     is_1d = X.ndim == 1
     X = check_array(X, dtype=np.float)
+    X = np.atleast_2d(X)
```

Comment:
@amueller Is this OK ? I can't ensure that X passed to this function is always 2D because in `logistic.py` a dot product is taken later and it complains about the dimension. Alternatively, we could make the argument 2D, before passing it to this function and reshape it after it returns.